.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn commitlint --edit 
```

Comment:
`npx --no -- commitlint --edit $1`

Reply:
What's the reason for this? Both should work fine.

Reply:
In the readme, we specify ```yarn``` as a package manager. so preferred yarn to add and run commit-msg hook locally

Reply:
I'm good with either. Just wanted to know the reasoning behind this to understand the motivation.

Reply:
any updates here @bumi @escapedcat. I was willing to get this pr merged so that newer changes in readme doesn't create any merge conflicts.

Reply:
I have no idea. I trust you. @dylancom why do you recommend `npx`?

Reply:
Both should work fine indeed, my comment was just about consistency. Let's update our pre-commit hook with yarn too then :)

Reply:
shall I change to npx then?. but in readme, we specify yarn as a package manager. we shall also mention there npx as well. if user use older npm version(they don't contain npx by default) or somehow npx doesnt work in system then it might be a problem.

Reply:
Let's go for Yarn, would you mind updating .husky/pre-commit?

Reply:
okay, I will update this in new PR!

Reply:
@bumi  kindly review and merge